kafka/sink/cmd/receive_adapter/main.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"knative.dev/eventing/pkg/adapter/v2"
+
+	kafkasinkadapter "knative.dev/eventing-contrib/kafka/sink/pkg/adapter"
+)
+
+func main() {
+	adapter.Main("kafkasink", kafkasinkadapter.NewEnvConfig, kafkasinkadapter.NewAdapter)
```

Comment:
Does it works on kube? `adapter.Main` requires a service account that can watch config maps and stuff, and from the yamls below it doesn't seem like you're using any

Reply:
It worked in k8s v1.18 - see example output in README.md